build_tools/requirements.txt

```diff
@@ -2,8 +2,8 @@ cython>=0.29.0
 esig==0.9.7
 fbprophet>=0.7.1
 hcrystalball>=0.1.9
-numba>=0.50,<0.53
-numpy==1.19
+numba>=0.53
```

Comment:
These are the minimal versions that support 3.9 that are needed for the build to pass. I can put an upper bound on numba if needed?

Reply:
hm, @mloning, why did we have those weird bounds on `numba` in the first place?
I'm sure there was a good reason, but I wasn't involved in setting this up.

Reply:
Better safe than sorry, let's check this twice.

Reply:
There may have been some incompatibility - I'm not exactly sure either.